demo/components/draw-line-text/your-text.tsx

```
@@ -1,43 +1,42 @@
-  const [inputValue, setInputValue] = useState('Write Anything')
-  const [key, setKey] = useState(0)
-  const debounceRef = useRef<NodeJS.Timeout | null>(null)
+    const [text, setText] = useState("Write Anything");
+    const [inputValue, setInputValue] = useState("Write Anything");
+    const [key, setKey] = useState(0);
+    const debounceRef = useRef<NodeJS.Timeout | null>(null);
 
-  useEffect(() => {
-    if (text == inputValue) return
+    useEffect(() => {
+        if (text == inputValue) return;
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)